bin/alert_on_unused_contexts.py

```diff
@@ -0,0 +1,81 @@
+import os
+import yaml
+
+from requests.exceptions import HTTPError, Timeout
+
+from services.github_service import GithubService
+from services.circleci_service import CircleciService
+from services.slack_service import SlackService
+
+# pylint: disable=W0718, R0914
+
+GITHUB_ORG = "ministryofjustice"
+
+
+def get_environment_variables() -> tuple:
+    slack_token = os.getenv("ADMIN_SLACK_TOKEN")
+    if not slack_token:
+        raise ValueError(
+            "The env variable ADMIN_SLACK_TOKEN is empty or missing")
+    github_token = os.getenv("ADMIN_GITHUB_TOKEN")
+    if not github_token:
+        raise ValueError(
+            "The env variable ADMIN_GITHUB_TOKEN is empty or missing")
+    circleci_token = os.getenv("ADMIN_CIRCLECI_TOKEN")
+    if not circleci_token:
+        raise ValueError(
+            "The env variable ADMIN_CIRCLECI_TOKEN is empty or missing")
+
+    return slack_token, github_token, circleci_token
+
+
+def get_all_pipeline_ids_for_all_repositories(repo_list, circle_ci_service):
+    all_pipeline_ids = []
+
+    for repo in repo_list:
+        pipelines = circle_ci_service.get_circleci_pipelines_for_repository(repo)
+        for pipeline in pipelines:
+            all_pipeline_ids.append(pipeline["id"])
+
+    return all_pipeline_ids
+
+
+def main():
+    slack_token, github_token, circleci_token = get_environment_variables()
+    github_service = GithubService(github_token, GITHUB_ORG)
+    circle_ci_service = CircleciService(circleci_token, os.getenv("CIRCLE_CI_OWNER_ID"), GITHUB_ORG)
```

Comment:
Should probably be getting the env var from the `get_environment_variables()` function 😜